pkg/providers/instancetype/suite_test.go

```diff
@@ -1648,7 +1679,46 @@ var _ = Describe("InstanceTypeProvider", func() {
 				Expect(it.Capacity.Pods().Value()).To(BeNumerically("==", 10))
 			}
 		})
-		It("should reserve ENIs when aws.reservedENIs is set and is used in max-pods calculation", func() {
+		It("should reserve ENIs when aws.reservedENIs is set and use it in max-pods calculation but ignore in kube-reserved memory calculation when AMI family is AL2", func() {
```

Comment:
Can we implement this for every AMI family, not just AL2 and Bottlerocket? Should be able to collapse this with a `DescribeTable`.